src/ott-analytics.js

```diff
@@ -243,11 +243,15 @@ export default class OttAnalytics extends BasePlugin {
     const request: RequestBuilder = OTTBookmarkService.add(this.config.serviceUrl, this.config.ks, bookMark);
     request.doHttpRequest().then(
       data => {
-        if (data === CONCURRENT) {
-          this._concurrentFlag = true;
-        } else {
-          this.logger.debug('Analytics event sent', bookMark);
-        }
+          const o = JSON.parse(data);
```

Comment:
add `try catch` around `JSON.parse`
If parse fails add `this.logger.debug('Analytics response parsing failed', data);`